### PR TITLE
Link issues to Sonar flag

### PR DESCRIPTION
**Pull request by Lauri-Nomme**
Introduce --linkIssues flag to create links to Sonar from reported issues.
This is needed to be able to easily navigate to the specific issues from report.

(cherry picked from commit 364925d0811887f92719ded77703d31484bc05e5)

### CODE REVIEW

**Comment by Lauri-Nomme** [comment]
@thecampagnards poke :)

File: index.js [hunk 1]
```diff
@@ -92,6 +95,17 @@ function logError(context, error){
     context, errorCode, errorMessage, errorResponseStatusCode, errorResponseStatusMessage,  errorResponseBody);  
 }
 
+
+const issueLink = argv.linkIssues == 'true' ?
+    (data, issue) =>
+        c => `<a href=\"${data.sonarBaseURL}/project/issues?id=${encodeURIComponent(data.sonarComponent)}&issues=${encodeURIComponent(issue.key)}&open=${encodeURIComponent(issue.key)}\">${c}</a>` :
```
Comment (thecampagnards) [inline index.js:101]: Is it needed to escape the ´"´ ?

**Comment by thecampagnards** [comment]
Thanks for your pr, I have just 2 questions

File: index.js (hunk 1 above)
Comment (Lauri-Nomme) [reply index.js]: added. tested against Version 8.9.7

**Pull request merged**